dashboard/app/models/levels/bubble_choice.rb

```diff
@@ -220,6 +220,9 @@ def setup_sublevels(sublevel_names)
     # otherwise, update sublevels to match
     levels_child_levels.sublevel.destroy_all
     Level.where(name: sublevel_names).each do |new_sublevel|
+      if new_sublevel.is_a?(BubbleChoice) ||  new_sublevel.is_a?(LevelGroup)
+        raise "BubbleChoice level #{name.dump} cannot contain #{new_sublevel.class.name} level #{new_sublevel.name.dump}"
```

Comment:
oh, I forgot about this rule! We'll also want to implement this as a validation

Reply:
I can change it to a validation, but doing so still raises and error, so there's no real advantage in this case: 
![Screen Shot 2022-05-19 at 5 48 37 PM](https://user-images.githubusercontent.com/8001765/169426952-ca80e84e-f660-4937-b592-c2b09d8c839f.png)

Reply:
Making it a validation enforced at the model layer rather than a manually-raised error enforced only by a helper method is an advantage in and of itself!

Reply:
I buy it! :-)